be/src/olap/task/engine_clone_task.cpp

```diff
@@ -827,4 +828,9 @@ Status EngineCloneTask::_finish_full_clone(Tablet* tablet,
     // TODO(plat1ko): write cooldown meta to remote if this replica is cooldown replica
 }
 
+std::string _mask_token(const std::string& str) {
```

Comment:
class name should be added. EngineCloneTask::_mask_token

Reply:
OK, fixed